api/rpc/dashboard/dashboard.go

```diff
@@ -60,6 +60,9 @@ func (s *Server) List(ctx context.Context, in *ListRequest) (*ListReply, error)
 	if err != nil {
 		return nil, convertError(err)
 	}
+	for _, dashboard := range dashboards {
+		dashboard.Data = "" //remove data, data are loaded one by one using get
```

Comment:
@freignat91 This line is causing the tests to fail. Tests need to be updated to reflect this modification.